### PR TITLE
Create dhruvakalur.json

### PR DESCRIPTION
**Pull request by dhruvakalur**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/dhruvakalur.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "description": "My Personal EMAIL",
+  "owner": {
+    "username": "dhruvakalur",
+    "email": "dhruvakalur9@gmail.com",
```
Comment (Stef-00012) [inline domains/dhruvakalur.json:5]: Remove trailing `,`

```suggestion
    "email": "dhruvakalur9@gmail.com"
```

File: domains/a.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "dhruvakalur",
+           "email": "",
+           "discord": "794802369216380949",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.n6sY7FH0bwRVpn5paxjZE9b0AS1bS9fiw6WV4sdfPhfnRrO9vR_Mxznkv_LzM5IWuOPUrNzJ8el1rUP1_BEnafofpoSYWV4J326U0TjFnhOysYjmpJ00oNE_TkBdF-gIlwZ3Ps6X2_i5OwLkWysa6oz7-DPHMA2bN27pC7VV7uMwt8gbvLH7BBgnpgZk7vf9eQ7hb8lgHDfjaeaxfG8PoGnO4SBMxcS_Q9hQxW-z7WsfBTTOeKC8jDg3ql5CCtw4epp61hmMGMD8Fk0ETZTDXVHLRVVLsR78uKwgRm-eifgDRVgycfoGfZpRkdZAWFG5fmAYTAxOxCFj6Efa44n-tA.PzULyP0rlpaGmeRh1tX5pA._J47HD2_nZv5f8j_vl5QZZCxbm6K4_nvrZ4LVcTdjH275xenN3SC3fbM-kQFokjiiKtiaSdZMxokzZtBbDBXHORYh2Z9Ve1ENytjWkmOP3U.YDrfLdRHD6-J3Bb9_7iysA"
+        },
+    
+        "record": {
+            "A": ["45.90.12.29"]
```
Comment (Stef-00012) [inline domains/a.json:10]: Can you show a preview of your website?

**Review by Stef-DP — DISMISSED** [review]

**Comment by dhruvakalur** [comment]
the server host i use says i must first set A record to their IP before proxying

File: domains/dhruvakalur.json (hunk 1 above)
Comment (dhruvakalur) [reply domains/dhruvakalur.json:5]: the server host i use says i must first set A record to their IP before proxying

45.90.12.29:7284 : preview

**Comment by dhruvakalur** [comment]
> > the server host i use says i must first set A record to their IP before proxying
> > 45.90.12.29:7284 : preview
> 
> do you have an image?

image

**Pull request merged**